onnxruntime/core/mlas/lib/x86_64/AssembleAvxVnni.h

```diff
@@ -0,0 +1,178 @@
+/*++
+
+Copyright (c) 2020 Intel Corporation. All rights reserved.
+
+Licensed under the MIT License.
+
+Module Name:
+
+    AssembleAvxVnni.h
+
+Abstract:
+
+    This module contains macros to build VNNI instructions for toolchains that
+    do not natively support this newer instruction set extension.
+
+--*/
+
+//
+// Map friendly register names to the encoded register index.
+//
+
+        .equ    .LYmmIndex_ymm0, 0
+        .equ    .LYmmIndex_ymm1, 1
+        .equ    .LYmmIndex_ymm2, 2
+        .equ    .LYmmIndex_ymm3, 3
+        .equ    .LYmmIndex_ymm4, 4
+        .equ    .LYmmIndex_ymm5, 5
+        .equ    .LYmmIndex_ymm6, 6
+        .equ    .LYmmIndex_ymm7, 7
+        .equ    .LYmmIndex_ymm8, 8
+        .equ    .LYmmIndex_ymm9, 9
+        .equ    .LYmmIndex_ymm10, 10
+        .equ    .LYmmIndex_ymm11, 11
+        .equ    .LYmmIndex_ymm12, 12
+        .equ    .LYmmIndex_ymm13, 13
+        .equ    .LYmmIndex_ymm14, 14
+        .equ    .LYmmIndex_ymm15, 15
+
+        .equ.   LXmmIndex_xmm0, 0
```

Comment:
The ".equ." here doesn't match the .equ from above. Can you fix this block?

Reply:
Fixed that.